sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/__init__.py

```diff
@@ -15,6 +15,7 @@
     PhoneNumberAssignmentType,
     PhoneNumberCapabilityType,
     PhoneNumberType,
+    PhoneNumberOperationStatus
```

Comment:
Why is this being made public?

Reply:
We're doing it to compare the result from the LRO directly from the PhoneNumberOperationStatus enum and to make sure the status that comes from the service stays consistent with the one we have registered.

Reply:
We should not make this part of the public API just to make the tests run....
We can import from the internal `_generated` namespace in the tests.

Reply:
@annatisch  / @jbeauregardb could we address this one as a separate thing ? It is blocking an unrelated project (chat) and we are stuck since Thursday here.